config/operator.yaml

```diff
@@ -59,8 +59,33 @@ spec:
             periodSeconds: 1
             successThreshold: 1
             timeoutSeconds: 15
+          securityContext:
+            runAsUser: 65532
+            runAsGroup: 65532
+            runAsNonRoot: true
+            readOnlyRootFilesystem: true
+            seccompProfile:
+              type: RuntimeDefault
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
           terminationMessagePolicy: FallbackToLogsOnError
+          volumeMounts:
+            - mountPath: /tmp
+              name: scratch
+              subPath: tmp
+      securityContext:
+        runAsUser: 65532
+        runAsGroup: 65532
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
```

Comment:
Kinda! You can set a cluster-wide default, but by default it's still unconfined.

This is the feature you're thinking of: https://kubernetes.io/docs/tutorials/security/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads

However, if this flag is not enabled, then you'll still get the previous behavior (Unconfined) so it's best to be explicit so that the behavior is consistent across clusters.